src/modules/cpu/kernel/spectrogram.hpp

```diff
@@ -0,0 +1,388 @@
+#include "rppdefs.h"
+#include "rpp_cpu_simd.hpp"
+#include "rpp_cpu_common.hpp"
+#include <chrono>
+#include <complex>
+
+inline Rpp32f reduce_add_ps1(__m256 src)
+{
+    __m256 src_add = _mm256_add_ps(src, _mm256_permute2f128_ps(src, src, 1));
```

Comment:
Change the naming convention for Registers
for __m256 / __m128 (float) pVarName
for __m256i / __m128i (int) pxVarName

Use camel case

Reply:
Done